tests/unit/v2/test_transformer.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+
+import ast
+from sagemaker.tools.compatibility.v2.ast_transformer import ASTTransformer
```

Comment:
the code has been moved a couple times, but now lives in `sagemaker.cli.compatibility.v2` (I think this should also fix the failing build)